pom.xml

```
@@ -54,7 +54,7 @@
             ${project.basedir}/../coverage-report/target/site/jacoco-aggregate/jacoco.xml
         </sonar.coverage.jacoco.xmlReportPaths>
 
-        <axon.version>4.9.0-SNAPSHOT</axon.version>
+        <axon.version>4.10.0-SNAPSHOT</axon.version>
```

We can switch this to `4.10.0` now :raised_hands: 
```suggestion
        <axon.version>4.10.0</axon.version>
```